src/style-spec/reference/v8.json

```diff
@@ -4450,6 +4450,66 @@
       },
       "property-type": "data-driven"
     },
+    "circle-ang-start": {
+      "type": "number",
+      "default": 0,
+      "minimum": 0,
+      "transition": true,
+      "units": "angle",
+      "doc": "Angle value to start drawing a circle.",
+      "sdk-support": {
+        "basic functionality": {
+          "js": "----",
+          "android": "----",
+          "ios": "----",
+          "macos": "----"
+        },
+        "data-driven styling": {
+          "js": "0.18.0",
+          "android": "5.0.0",
+          "ios": "3.5.0",
+          "macos": "0.4.0"
+        }
+      },
+      "expression": {
+        "interpolated": true,
+        "parameters": [
+          "zoom",
+          "feature"
+        ]
+      },
+      "property-type": "data-driven"
+    },
+    "circle-ang-end": {
+      "type": "number",
+      "default": 360,
+      "minimum": 0,
+      "transition": true,
+      "units": "angle",
+      "doc": "Angle value to finish drawing a circle.",
+      "sdk-support": {
+        "basic functionality": {
+          "js": "----",
+          "android": "----",
+          "ios": "----",
+          "macos": "----"
+        },
+        "data-driven styling": {
+          "js": "0.18.0",
+          "android": "5.0.0",
+          "ios": "3.5.0",
+          "macos": "0.4.0"
+        }
```

Comment:
data-driven styling key needs to be updated to be accurate

Reply:
Not fully clear to what value I should update. The minor versions?

Reply:
Remove all keys except for js, and then if on js, if the property supports evaluating data expressions (like `['get', 'foo']`), then set the js value to the next release version ("1.10.0").